ppa/archive/tests/test_gale.py

```diff
@@ -15,22 +16,32 @@
 @override_settings()
 def test_get_local_ocr(tmp_path):
     item_id = "CB0123456789"
-    page_num = "0001"
-    content = "Testing...\n1\n2\n3"
+    content = {"0001": "Testing...\n1\n2\n3"}
     # Mock ocr files for testing
-    ocr_dir = tmp_path.joinpath("147", item_id)
-    ocr_dir.mkdir(parents=True)
-    ocr_file = ocr_dir.joinpath(f"{item_id}_{page_num}0.txt")
-    ocr_file.write_text(content)
+    ocr_dir = tmp_path / "147"
+    ocr_dir.mkdir()
+    ocr_file = ocr_dir / f"{item_id}.json"
+    with ocr_file.open("w") as outfile:
```

Comment:
I don't know if we need to be defensive here, but the default encoding is platform dependent and not necessarily "utf-8 (although usually it will be).

```suggestion
    with ocr_file.open("w", encoding="utf-8") as outfile:
```